### PR TITLE
Describe HtmlEditor Table Resizing Properties

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: api-reference/10 UI Components/dxHtmlEditor/1 Configuration/tableResizing.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ inheritsType: dxHtmlEditorTableResizing
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures table resizing.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Components/dxHtmlEditor/1 Configuration/tableResizing.md]: -ing

File: api-reference/_hidden/dxHtmlEditorTableResizing/enabled.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Enables table resizing.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/dxHtmlEditorTableResizing/enabled.md]: -ing

File: api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md [hunk 3]
```diff
@@ -5,7 +5,8 @@ default: 40
...
 ---
 ##### shortDescription
-<!-- Description goes here -->
+##### shortDescription
+The minimum column width.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+When the content width exceeds this property's value and it is impossible to move the content to a new line, the actual column width equals the content width.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md]: Does this sentence describe the case when a user wants to shrink a column down to the minColumnWidth value, but the column won't let him?

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md (hunk 3 above)
Comment (AlexJameson) [reply api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md]: Yes, exactly.

File: api-reference/_hidden/dxHtmlEditorTableResizing/enabled.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Allows you to resize tables.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/dxHtmlEditorTableResizing/enabled.md]:
```suggestion
Enables users to resize tables.
```

File: api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md (hunk 3 above)
Comment (RomanTsukanov) [reply api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md]: Then it's unclear that you mean this case. Please rewrite.

File: api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ default: 40
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The minimum column width.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+When the content width exceeds this property's value and users resize the column, the minimum column width equals to the content width if it is impossible to break content into lines.
```
Comment (RomanTsukanov) [inline api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md]: Add an example when required.

```suggestion
When a column's content exceeds **minColumnWidth**, and it is impossible to break the content into several lines, the minimum column width equals the content width. For example, if **minColumnWidth** is 50 but a column's content requires 60 pixels, users can narrow the column down to 60 pixels only.
```

**Review by alexander-kotov-dx — APPROVED** [review]

File: api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ default: 40
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The minimum column width.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+When a column's content exceeds **minColumnWidth**, and it is impossible to break the content into several lines, the minimum column width equals the content width. For example, if **minColumnWidth** is 50 but a column's content requires 60 pixels, users can narrow the column down to 60 pixels only.
```
Comment (albertov05) [inline api-reference/_hidden/dxHtmlEditorTableResizing/minColumnWidth.md]:
```suggestion
When a column's content exceeds **minColumnWidth**, and it is impossible to break the content into several lines, the minimum column width equals the content width. For example, if **minColumnWidth** is 50 pixels but a column's content requires 60 pixels, users can decrease the column width to 60 pixels only.
```

File: api-reference/_hidden/dxHtmlEditorTableResizing/minRowHeight.md [hunk 7]
```diff
@@ -5,7 +5,7 @@ default: 24
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+The minimum row height.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+When the content height exceeds this property's value and users resize the row, the minimum row height equals to the content height.
```
Comment (albertov05) [inline api-reference/_hidden/dxHtmlEditorTableResizing/minRowHeight.md]:
```suggestion
When the content height exceeds this property's value and users resize the row, the minimum row height equals the content height.
```

**Pull request merged**